driver/src/main/java/module-info.java

```diff
@@ -35,7 +35,7 @@
     requires io.netty.buffer;
     requires io.netty.codec;
     requires io.netty.resolver;
-    requires transitive java.logging;
+    requires static java.logging;
```

Comment:
This change triggers compilation failure, caused by the following warning: `class java.util.logging.Level in module java.logging is not indirectly exported using requires transitive`.

Reply:
Wow interesting. I tried creating a reproducer locally but wasn't able to.

According to what I've seen online we can make it `static` and `transitive`, but I'm not 100% sure. In this case it's true, the logging class is exporting JUL to downstream users (using the logger in the public return types).

Reading through https://nipafx.dev/java-modules-implied-readability/ right now.

Reply:
Is it still failing for the same reasons?